modules/payments/modules/payments/api.js

```diff
@@ -0,0 +1,42 @@
+import { getGlobalOptions } from "@options";
+
+
+const global = getGlobalOptions(); 
+const BASE_URL = global.url; // change your BASE_URL in `options/options.js` to edit this value
+const token = "Token 676364403988909cde7f501aa2efeaf9ca30d18c"
```

Comment:
This should be a module option

Reply:
Ignore the above comment, this one will be stored in encrypted-storage most likely. I'll update you on this